ci-operator/step-registry/upi/vsphere/platform-none/ovn/pre/upi-vsphere-platform-none-ovn-pre-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: upi-vsphere-platform-none-ovn-pre
+  steps:
+  - ref: ipi-install-rbac
+  - chain: upi-conf-vsphere-platform-none-ovn
```

Comment:
`upi-conf-vsphere-platform-none-ovn` is the chain used currently in platform-none pre-chain.